.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  name: Lint
+
+  runs-on: ubuntu-latest
+
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: lint
+      run: |
+        go get -u honnef.co/go/tools/cmd/staticcheck@latest &&
+        $HOME/go/bin/staticcheck
+
+jobs:
+  name: Test
+
+  runs-on: ubuntu-latest
+
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: test
+      run: |
+        go generate &&
+        make
```

Comment:
It looks like `make` without additional args, will run all the steps: `test vet lint check-gofmt build`